.credo.exs

```diff
@@ -9,7 +9,10 @@
       checks: [
         {Credo.Check.Consistency.TabsOrSpaces},
         {Credo.Check.Design.AliasUsage, false},
-        {Credo.Check.Readability.MaxLineLength, priority: :low, max_length: 100}
+        {Credo.Check.Readability.MaxLineLength, priority: :low, max_length: 100},
+        {Credo.Check.Readability.PipeIntoAnonymousFunctions, false},
+        # This rule incorrectly flags uses of Witchcraft.Semigroupoid.apply/2
+        {Credo.Check.Refactor.Apply, false}
```

Comment:
Hmm I wonder if we should rework the relevant parts of the library to make them compatible with the updated community naming conventions 🤔 I'll open an Issue at minimum.